main.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/rkoesters/xdg/basedir"
-	todoist "github.com/sachaos/todoist/lib"
+	x"github.com/sachaos/todoist/lib"
```

Comment:
this last commit broke the build. removing this change works fine.

Reply:
Resolved. This is what happens when I write code outside of my tooling, go figure.